### PR TITLE
Added panel's Documentation

### PR DESCRIPTION
**Pull request by Bumblebee-3**

### CODE REVIEW

**Review by Leref — CHANGES_REQUESTED** [review]
All of this must follow the syntax methods for AoiEngine to read.

title, description, and id are missing for each.

**Comment by Bumblebee-3** [comment]
Have updated it.

File: src/guides/@akarui/aoi.panel/advanced.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+title: akarui/aoi.panel
```
Comment (Leref) [inline src/guides/@akarui/aoi.panel/advanced.md:2]: it doesn't need `akarui/` only the name.

**Review by Leref — CHANGES_REQUESTED** [review]

File: src/guides/@akarui/aoi.panel/advanced.md (hunk 1 above)
Comment (Bumblebee-3) [reply src/guides/@akarui/aoi.panel/advanced.md]: Have updated, Pls check

Comment (Leref) [reply src/guides/@akarui/aoi.panel/advanced.md]: The actual name syntax isn't updated.

File: src/guides/akarui/aoi.panel/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+---
+title: akarui/aoi.panel
```
Comment (Leref) [inline src/guides/akarui/aoi.panel/README.md:2]: it should be aoi.panel.

File: src/guides/akarui/aoi.panel/advanced.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
+---
+title: akarui/aoi.panel
```
Comment (Leref) [inline src/guides/akarui/aoi.panel/advanced.md:2]: Remove akarui/aoi.panel and rename to according information title.

File: src/guides/akarui/aoi.panel/funcs.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+---
+title: akarui/aoi.panel
```
Comment (Leref) [inline src/guides/akarui/aoi.panel/funcs.md:2]: From previous comment.

File: src/guides/akarui/aoi.panel/panel.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
+---
+title: akarui/aoi.panel
```
Comment (Leref) [inline src/guides/akarui/aoi.panel/panel.md:2]: As well from previous comment.

**Review by Leref — CHANGES_REQUESTED** [review]
Must be changed before accepting.

**Comment by Bumblebee-3** [comment]
Have updated the docs, with Colours and code editor's info

**Review by Leref — APPROVED** [review]

**Pull request merged**